### PR TITLE
Update AggregateMessageOrigin

### PR DESCRIPTION
**Pull request by claravanstaden**
Resolves: [SNO-747]( https://linear.app/snowfork/issue/SNO-747)

Polkadot-sdk companion: https://github.com/Snowfork/polkadot-sdk/pull/38

### CODE REVIEW

File: relayer/contracts/beefy_client.go [hunk 1]
```diff
@@ -72,12 +72,13 @@ type BeefyClientValidatorSet struct {
 
 // Uint16ArrayArray is an auto generated low-level Go binding around an user-defined struct.
 type Uint16ArrayArray struct {
-	Data []*big.Int
+	Data   []*big.Int
+	Length *big.Int
```
Comment (claravanstaden) [inline relayer/contracts/beefy_client.go:76]: Unrelated to this PR but I think this comes from `./scripts/init.sh` or `start-services.sh`

**Comment by yrong** [comment]
@claravanstaden Just made some fixes for the compile error and failed tests, changes mainly in https://github.com/Snowfork/polkadot-sdk/pull/39/ based on, hope it's helpful and happy weekend :relaxed:

**Comment by claravanstaden** [comment]
@yrong thank you, I appreciate it! 😄

File: relayer/contracts/beefy_client.go (hunk 1 above)
Comment (alistair-singh) [reply relayer/contracts/beefy_client.go:76]: Yeah this was that change I did to add length to UInt16Array in the solidity and start-services.sh rebuilt the contracts.

**Review by vgeddes — APPROVED** [review]
+1

**Review by alistair-singh — APPROVED** [review]

**Pull request merged**